libiocage/lib/Jails.py

```diff
@@ -1,12 +1,13 @@
-import re
+from typing import Generator, Union, Iterable
 
 import libzfs
 
 import libiocage.lib.Jail
+import libiocage.lib.JailFilter
 import libiocage.lib.helpers
```

Comment:
Are you running isort like CONTRIBUTING says? These look out of order, I'm seeing that around the files here in general

Reply:
Didn't care about the order yet. Let's put in a static analyzer rule in addition to the contribution guidelines.

I suggest to tackle those kind of changes in a separate PR, so that we concentrate on a specific feature in a PR. Too often features have implications to other parts of libiocage as well. But I'm pretty sure this will change in the near future after the interfaces are well tested.

https://github.com/iocage/libiocage/issues/47 is the ticket for that